doc/source/configuration/magnum-capi.rst

```diff
@@ -105,16 +105,10 @@ Next, copy the CAPI management cluster's kubeconfig file into your stackhpc-kayo
 
 The following config should also be set in your stackhpc-kayobe-config environment:
 
-.. code-block:: ini
-    :caption: magnum.conf
-
-    [nova_client]
-    endpoint_type = publicURL
```

Comment:
Does it require a new magnum-capi-helm version?

Reply:
Yes it will, marking as draft until that release is ready and container images are available.

Reply:
magnum-capi-helm version bumped in https://github.com/stackhpc/stackhpc-kayobe-config/pull/1007